pkgs/development/python-modules/django-cacheops/default.nix

```diff
@@ -1,27 +1,41 @@
 { lib
 , buildPythonPackage
 , fetchPypi
+, fetchpatch
 , django
 , funcy
 , redis
-, pytest-django
+, six
 , pytestCheckHook
+, pytest-django
+, mock
+, dill
+, jinja2
+, before_after
 , pythonOlder
-, six
+, pkgs
```

Comment:
why use `pkgs.redis` instead of `redis`?

Reply:
because redis in this context refers to python3.pkgs.redis, instead of the redis server package